### PR TITLE
Toponaming fix dress up shadow subs

### PR DESCRIPTION
**Pull request by bgbsww**
This should solve many "invalid edge" warnings on chamfers and fillets.

### CODE REVIEW

File: src/Mod/PartDesign/Gui/TaskDressUpParameters.cpp [hunk 1]
```diff
@@ -140,6 +140,39 @@ void TaskDressUpParameters::referenceSelected(const Gui::SelectionChanges& msg,
...
+    if (!base) {
+        return;
+    }
+    int count = Part::Feature::getTopoShape(base).countSubShapes(TopAbs_EDGE);
+    auto subValues = pcDressUp->Base.getSubValues(false);
+    std::size_t len = subValues.size();
+    std::string name("Edge");
+    for (int i = 0; i < count; ++i) {
+        name.resize(4);
+        name += std::to_string(i + 1);
```
Comment (kadet1090) [inline src/Mod/PartDesign/Gui/TaskDressUpParameters.cpp:159]: Out of curiosity - why not use string stream like it is done in the non-tnp fix code?

```
    for (int i = 0; i < count; i++){
        std::ostringstream ss;
        ss << "Edge" << ii+1;
        
        auto name = ss.str();
```

Or at least

```
std::string name = "Edge" + std::to_string(i+1);
```

Current solution with reusing the same variable, resizing it etc is very hard to follow and understand. 

Of course it would be even better to use something like `std::format` (if only we could...)

Comment (bgbsww) [reply src/Mod/PartDesign/Gui/TaskDressUpParameters.cpp:159]: Good point.  Because I was focused on finding the right missing code from the LS3 side to bring over, and not on the modernization of same.
